src/region.js

```diff
@@ -315,42 +236,39 @@ var Region = MarionetteObject.extend({
     if (view.destroy) {
       view.destroy();
     } else {
+      const shouldTriggerDetach = !!view._isAttached;
```

Comment:
I think it might be nice stylistically to move everything in this else to a method that helps document the functionality.. like `_destroyBackboneView` or something